guides/hosting/installation-updates/cluster-setup.md

```diff
@@ -67,6 +67,16 @@ Updates of such systems require a certain amount of effort, as issues often aris
 
 For obtaining security fixes, without version upgrades, we provide a dedicated [Security plugin](https://store.shopware.com/swag136939272659f/shopware-6-sicherheits-plugin.html). This is compatible with all Shopware versions and corresponding hot fixes are only included in versions that are affected.
 
+### Update of composer dependencies
+
+It may happen that third-party dependencies are affected by security vulnerabilities. In that case, a new shopware version will be released with a version of the dependency where the security vulnerability is already fixed. If an update to the latest shopware version in a timely manner is not possible, it is recommended to update the affected dependency manually. This can be done by using the following command:
```

Comment:
```suggestion
To ensure the security of your Shopware installation, it's essential to be vigilant about third-party dependencies that might be affected by security vulnerabilities. In that case, a new Shopware version will be released with updated dependencies. If an update to the latest Shopware version in a timely manner is not possible, it is recommended to update the affected dependency manually. This can be done by using the following command:
```